### PR TITLE
Created note deletion page ui, fixed issues with code

### LINKED ISSUE #4: Create note deletion page UI

### PR DESCRIPTION
**Pull request by alan-thomas-shaji**
Fixes #4, fixed issues with #10 
Screenshot 2022-09-22 at 9 01 42 PM

### CODE REVIEW

**Comment by alan-thomas-shaji** [comment]
Got an eslint error in pre-commit hook like so:
Screenshot 2022-09-22 at 8 56 46 PM
But trying to change the order made further errors pop up which didn't make quite sense, so used `eslint-disable-next-line import/order` to bypass that.

**Comment by yedhink** [comment]
>But trying to change the order made further errors pop up which didn't make quite sense

Can you create a video and show me what all were the other errors? Thanks

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -1,40 +1,27 @@
...
+  const handleDelete = noteId => {
+    setNotes(oldNotes => oldNotes.filter(note => note.id !== noteId));
+    setShowDeleteAlert(false);
   };
 
   return (
     <Alert
       isOpen
-      isSubmitting={deleting}
       message="Are you sure you want to continue? This cannot be undone."
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:19]: Refer the Slack thread.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 2]
```diff
@@ -3,38 +3,40 @@ import React from "react";
...
       <Dropdown buttonStyle="text" icon={MenuVertical}>
-        <li>Edit</li>
-        <li>Delete</li>
+        <li> Edit </li>
+        <li
+          onClick={() => {
+            setShowDeleteAlert(true);
+            setSelectedNoteId(note.id);
+          }}
+        >
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:21]: Refer Slack thread.

**Comment by alan-thomas-shaji** [comment]
https://www.loom.com/share/34de4e21cafd4a7f99a7761a55ad869e

**Review by yedhink — COMMENTED** [review]
@alan-thomas-shaji

**Pull request merged**